### DISCUSSION TITLE
[Support] 参数校验的报错信息里提示的参数名不对。 (@midwayjs/validate)

### DISCUSSION THREAD

**Discussion by googoo625** [midwayjs/midway#4000] [Q&A]
### Describe the problem（描述问题）

接口定义如下，使用了@Valid ：
~~~
async getByParent(@Valid(RuleType.string().required()) @Query("parent") parent: string) {
  //
}
~~~

调用时若无parent参数，则有校验报错是： MidwayValidationError: "value" 是必须的
这里参数名不对，按说应该是："parent" 是必须的。

请教该如何修改？

### Midway Versions（Midway 版本）

"@midwayjs/validate": "3.16.8",

**Comment by waitingsong** [comment]
把 “parent” 换成其它名字正常不？

**Comment by czy88840616** [comment]
是这个问题吧。https://github.com/midwayjs/midway/issues/3092

**Reply by czy88840616** [reply]
在 TS 以及 JS 标准中，参数装饰器目前均无法获取到参数名。

**Reply by googoo625** [reply]
是的，照其添加 label，问题解决。感谢。
~~~
async getByParent(@Valid(RuleType.string().required().label("parent")) @Query("parent") parent: string) {
  //
}
~~~

组件能够自动提取出参数名吗？

**Reply by czy88840616** [reply]
语言本身限制，取不出来呢

**Comment by googoo625** [comment]
添加 label 即可。
